Documentation/Upgrade/rook-upgrade.md

```diff
@@ -49,16 +50,18 @@ those releases.
 ## Breaking changes in v1.13
 
 * The minimum supported version of Kubernetes is v1.23.
-* Support for the admission controller/webhooks has been removed. If admission controller/webhooks is enabled, disable by changing
-`ROOK_DISABLE_ADMISSION_CONTROLLER: "true"` in operator.yaml before upgrading to rook v1.13. CRD validation is now enabled with [Common Expression Language](https://kubernetes.io/docs/reference/using-api/cel/). This requires Kubernetes version 1.25 or higher.
-
-## Breaking changes in v1.12
-
-* The minimum supported version of Kubernetes is v1.22.
+  Upgrade to Kubernetes v1.23 or higher before upgrading Rook.
+* The minimum supported version of Ceph is v17.2.0.
+  If a lower version is currently deployed, [Upgrade Ceph](./ceph-upgrade.md) before upgrading Rook.
 * CephCSI CephFS driver introduced a breaking change in v3.9.0. If any existing CephFS storageclass in
   the cluster has `MountOptions` parameter set, follow the steps mentioned in the
   [CephCSI upgrade guide](https://github.com/ceph/ceph-csi/blob/v3.9.0/docs/ceph-csi-upgrade.md/#upgrading-cephfs)
   to ensure a smooth upgrade.
+* Support for the admission controller has been removed. CRD validation is now enabled with
+  [Validating Admission Policies](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/).
+  This requires Kubernetes version v1.25 or higher.
+  If the admission controller is enabled, upgrade to Kubernetes v1.25 or higher before upgrading Rook.
+  It has been disabled by default since Rook v1.10.9. For more info, see https://github.com/rook/rook/pull/11532.
```

Comment:
After some thought, it seemed like a better suggestion to me to recommend that any current admission users upgrade to k8s v1.25+ rather than to set `ROOK_DISABLE_ADMISSION_CONTROLLER: true`. Changing the setting would only affect the current v1.12 deployment, and it would result in a short "continuity loss" for the underlying feature which I would call "complex CRD validations."

For users who care about this, I think they will be better supported.

For users who don't (which we expect to be 95%+), it's a moot point.

Reply:
What if we just be clear that lower versions will ignore the admission policies? They can decide then if they want to upgrade K8s, no need to state that they really should. IMO line 63 is unnecessary, but not a big deal.

Reply:
I think we maybe should do both, but I'm a cynic. I've seen too many users do something that is against their own best interests because they failed to fully process through the impacts of a statement like "this feature is ignored for versions below X." Having a specific suggestion like "if you rely on this, don't upgrade before you have upgraded to at least X" seems like it would help solidify things for readers who are less "active."